src/encoded/schemas/gene.json

```diff
@@ -77,7 +159,57 @@
                 "description": "A unique identifier from external resource (e.g. HGNC, MGI, FlyBase, WormBase, ENSEMBL, MIM, UniProtKB, Vega, miRBase).",
                 "comment": "Submit as database_name:id. See changelog for potential databases.",
                 "type":  "string",
-                "pattern": "^((HGNC:\\d+)|(MGI:\\d+)|(FlyBase:FBgn\\d+)|(WormBase:WBGene\\d+)|(ENSEMBL:ENS[A-Z]*G\\d+)|(MIM:\\d+)|(UniProtKB:[0-9A-Z]+)|(Vega:OTT[A-Z]+G\\d+)|(miRBase:MI\\d+)|(IMGT/GENE-DB:[0-9A-Za-z/\\(\\)\\-]+)|(RefSeq:[A-Z]{2}_\\d+))$"
+                "pattern": "^((HGNC:\\d+)|(MGI:\\d+)|(FlyBase:FBgn\\d+)|(WormBase:WBGene\\d+)|(ENSEMBL:ENS[A-Z]*G\\d+)|(MIM:\\d+)|(UniProtKB:[0-9A-Z]+)|(Vega:OTT[A-Z]+G\\d+)|(miRBase:MI\\d+)|(IMGT/GENE-DB:[0-9A-Za-z/\\(\\)\\-]+)|(RefSeq:[A-Z]{2}_[\\d.]+))$"
+            },
+            "permission": "import_items"
+        },
+        "locations": {
+            "title": "Gene location coordinates",
+            "description": "Genomic locations using 1-based, closed coordinates for different assembles of this gene.",
+            "type": "array",
+            "default": [],
+            "uniqueItems": true,
+            "items": {
+                "title": "Gene location coordinate",
+                "description": "Genome position information for one assembly of this gene.",
+                "type": "object",
```

Comment:
I would go with title "Gene location", otherwise you need to say "coordinate" but each location has two coordinates?
For description I would go with: "Genomic position information of the gene for a specific genome assembly"

Reply:
MAybe: "Gene location specified using 1-based, closed coordinates for a specific version of the reference genome assembly."